test/core/models/ocelot/stanzas/InstructionSpec.scala

```diff
@@ -30,31 +30,22 @@ class InstructionSpec extends WordSpec with Matchers {
     val link = Link(linkId, linkDest, "", false)
```

Comment:
Might be good to have some further tests for button links as we have some new functionality.

Reply:
there are many test, but have been included in the ui building tests in the viewer code